tasks/1_golf-score/golf-score.js

```diff
@@ -0,0 +1,27 @@
+export const GOLF_SCORES = {
+  HOLE_IN_ONE: 'Hole-in-one!',
+  EAGLE: 'Eagle',
+  BIRDIE: 'Birdie',
+  PAR: 'Par',
+  BOGEY: 'Bogey',
+  DOUBLE_BOGEY: 'Double Bogey',
+  GO_HOME: 'Go Home!',
+};
+
+export const golfScore = (par, strokes) =>{
+  if ( strokes === 1) {
+    return GOLF_SCORES.HOLE_IN_ONE;
+  } else if ( strokes <= par -2 ) {
+    return GOLF_SCORES.EAGLE;
+  } else if (strokes === par -1) {
+    return GOLF_SCORES.BIRDIE;
+  } else if ( strokes === par ) {
+    return GOLF_SCORES.PAR;
+  } else if ( strokes === par +1) {
+    return GOLF_SCORES.BOGEY;
+  } else if ( strokes === par +2) {
+    return GOLF_SCORES.DOUBLE_BOGEY;
+  } else if ( strokes >= par +3) {
+    return GOLF_SCORES.GO_HOME;
+  }
```

Comment:
As a lot `if` statement can create not very readable code here is possible to rewrite to `switch` with counting difference on top, check for edge case with `HOLE_IN_ONE` and then in `switch` return correct result.